tests/test_monitors.py

```diff
@@ -91,6 +97,69 @@ def test_item_count_monitor_extracted_less_than_expected(make_data, item_count_s
     )
 
 
+def test_needs_to_configure_expected_item_count_monitor(make_data, expected_item_count_suite):
+    """Should raise an exception when ExpectedItemCountMonitor it's not configured """
+    data = make_data({'SPIDERMON_EXPECTED_ITEMS_THRESHOLD': 1.1})
+    runner = data.pop("runner")
+    with pytest.raises(NotConfigured):
+        runner.run(expected_item_count_suite, **data)
+
+    data = make_data({'SPIDERMON_EXPECTED_ITEMS_THRESHOLD': -0.1})
+    runner = data.pop("runner")
+    with pytest.raises(NotConfigured):
+        runner.run(expected_item_count_suite, **data)
+
+
+def test_expected_item_count_monitor_less_than_expected(make_data, expected_item_count_suite):
+    """ExpectedItemCountMonitor should fail when the desired # of items is not extracted """
+    data = make_data({'SPIDERMON_EXPECTED_ITEMS_THRESHOLD': 0.1})
+    runner = data.pop("runner")
+    data["stats"]["item_scraped_count"] = 10
+    data["stats"]["spidermon/expected_item_scraped_count"] = 20
+    runner.run(expected_item_count_suite, **data)
+    for r in runner.result.monitor_results:
+        assert "Extracted 10 items, expected at least 18" in r.error
+
+
+def test_expected_item_count_monitor_more_than_expected(make_data, expected_item_count_suite):
```

Comment:
Notice how you repeat the code, I would use `@pytest.parametrize` like here https://github.com/scrapinghub/arche/blob/60c8aa87a67b432ec1fa0c9690d43684373b5d53/tests/tools/test_api.py#L7 instead of having these 4 separate repeating tests.